server/src/db/migrations/20210101210207_createUsers.cjs

```diff
@@ -15,6 +15,7 @@ exports.up = async (knex) => {
     console.log(`Creating ${tableName}`);
     return knex.schema.createTable(tableName, (table) => {
       table.bigIncrements("id");
+      table.string("userName").notNullable().unique();
```

Comment:
this migration file has already been run when the app was first created, and we should not modify existing migrations files. Best practice is to make a new migration file for any new changes we want to make to the tables in our database. By directly editing this file means that if this were merged in, every group member would have to drop the database, re-create, and run the migrations again. Where as if we create a new migration file, then when the work is merged in group members just need to run the latest migration.